spec/API_specification/creation_functions.md

```diff
@@ -289,6 +289,37 @@ Returns evenly spaced numbers over a specified interval.
 
     -   a one-dimensional array containing evenly spaced values.
 
+(function-meshgrid)=
+### meshgrid(*arrays, /, indexing='xy')
```

Comment:
This signature was using `/, *, ` before. Shouldn't `indexing` be keyword-only here?

Reply:
Indeed. Updated.